doc/pages/communities.json

```diff
@@ -82,5 +82,17 @@
     "link":"http://emotionsanonymous.org/",
     "tags":["free", "support_groups"],
     "languages":["en"]
+  },
+  {
+    "name":"In the Wake of Suicide...trying to understand",
```

Comment:
I would put these two resources under Education, since Communities is more for resources that are support groups or meetups :) These are great!